vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -116,6 +118,11 @@ type refCount struct {
 type RefCountsMap struct {
 	refMap map[string]*refCount // Map of refCounts
 	mtx    *sync.RWMutex        // Synchronizes RefCountsMap ops
+
+	refcntInitSuccess bool // save refcounting success
+	isDirty           bool // flag to check reconciling has been interrupted
+
```

Comment:
nit  - I don't think the empty line is needed as the mutex protects the two boolean and can be grouped together. Format/nit only